js/pages.js

```diff
@@ -0,0 +1,128 @@
+// Fetch page data from WordPress
+function getPageData(page) {
+	const apiURL = `https://wp.iftheselandscouldtalk.org/wp-json/wp/v2/pages?slug=${page}&acf_format=standard`;
+
+	fetch(`${apiURL}`, {
+		method: "GET",
+		headers: {
+			"Content-Type": "application/json",
+		},
+	})
+		.then(response => response.json())
+		.then(response => {
+			// Page data
+			const data = response[0].acf;
+			// Hero
+			const hero = document.getElementById("hero"); // For hero section
+			const heroHeading = document.getElementById("hero-heading"); // For h1 in the hero
+			const heroSubheading = document.getElementById("hero-subheading");
+			const heroImage = document.getElementById("hero-image");
+			const heroButton1 = document.getElementById("hero-button-1");
+			const heroButton2 = document.getElementById("hero-button-2");
+
+			if (data.hero_image.url) {
+				appendData(data.hero_image, "img", "image", heroImage);
+				heroImage.setAttribute("src", data.hero_image.url);
+				data.hero_image.alt
+					? heroImage.setAttribute("alt", data.hero_image.alt)
+					: heroImage.setAttribute("alt", "If These Lands Could Talk");
+				hero.style.backgroundImage = `url(${data.hero_image.url})`;
+			}
+			data.hero_heading
+				? appendData(data.hero_heading, "h1", "text", heroHeading)
+				: null;
+			data.hero_subheading
+				? appendData(data.hero_subheading, "h2", "text", heroSubheading)
+				: null;
+			data.hero_button_1
+				? appendData(data.hero_button_1, "hero-button-1", "link", heroButton1)
+				: null;
+			data.hero_button_1
+				? appendData(data.hero_button_2, "hero-button-2", "link", heroButton2)
+				: null;
+
+			// Main Content Sections
```

Comment:
This is used on about half of the pages - those with a big single area for content like About Us.

Reply:
<img width="1523" alt="Screen Shot 2024-02-02 at 12 19 15 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/a0e7f119-6734-4390-bff1-267eced546dc">